### DISCUSSION TITLE
Running Rook Ceph 1.13.3 in an all VMware Rook Cluster with Fibre Channel Storage for OSD

### DISCUSSION THREAD

**Discussion by bdpearman** [rook/rook#13928] [Q&A]
I am trying to to get rook ceph 1.13.3 in an all VMware rook ceph cluster with a flash SAN as the backing for the OSDS.
I know that vmware and SANs are not recommended for rook ceph, or at least that's the impression I got.
I can not provide actual logs, but I can look in logs and respond with info here.

So far I have tried the following:

1. SAN storage as regular datastore with vmdk.
2. SAN storage as RBD device.
3. SAN as direct passthrough PCI device (this seemed to work, but unfortunately even if it does I can't use this method due to hardware limitations)

If anyone has used both VMware and rook ceph and gotten rook ceph to work, will you please provide me with all the info, details, and configs that I should be doing/setting to get this setup to work.

My OSDs are larger than 8T, but I can always change that if needed.  The total storage is between 50 and 100T.  Again, for reasons I can't go into, I can't provide exact numbers only approximations.

**Comment by subhamkrai** [comment]
@bdpearman I'm not sure about VMWware flash SAN but  interested to know about `I know that vmware and SANs are not recommended for rook ceph, or at least that's the impression I got.` from where you got this impression.
